classes/PodsData.php

```diff
@@ -3183,12 +3183,7 @@ public function traverse_recurse( $traverse_recurse ) {
 
 		$field = $traverse_recurse['fields'][ $traverse_recurse['depth'] ];
 
-		$ignore_aliases = array(
-			'wpml_languages',
-			'polylang_languages',
-		);
-
-		$ignore_aliases = apply_filters( 'pods_data_traverse_recurse_ignore_aliases', $ignore_aliases, $field, $traverse_recurse, $this );
+		$ignore_aliases = apply_filters( 'pods_data_traverse_recurse_ignore_aliases', [], $field, $traverse_recurse, $this );
```

Comment:
Need phpdoc, I know we didn't have any before but if we're adjusting then let's do that

Reply:
Can you provide the idea behind this filter? I merely moved compatibility code, no idea what this filter is for..

Reply:
Pods may see wpml_languages or polylang_languages aliases in the queries and attempt to automatically treat them like relationship traversal. We are just excluding them to prevent any unintentional traversals on those aliases.